client/settings/customization-option-notice/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import { __ } from '@wordpress/i18n';
+import interpolateComponents from 'interpolate-components';
+import { Icon, closeSmall } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import UpeContext from '../upe-toggle/context';
+
+const NoticeWrapper = styled.div`
+	background: #def1f7;
+	box-shadow: inset 4px 0px 0px #00aadc;
+	display: flex;
+	flex-flow: row;
+	margin-top: 16px;
+	padding: 16px 16px 16px 24px;
+`;
+
+const MessageWrapper = styled.span`
+	margin-right: 24px;
+`;
+
+const IconWrapper = styled.span`
+	cursor: pointer;
+	flex: 1 0 auto;
+`;
+
+const CustomizationOptionNotice = () => {
+	const {
+		dismissCustomizationOptionsNotice,
+		isUpeEnabled,
+		shouldShowCustomizationOptionsNotice,
+	} = useContext( UpeContext );
+
+	if ( ! isUpeEnabled || ! shouldShowCustomizationOptionsNotice ) {
+		return null;
+	}
+
+	return (
+		<NoticeWrapper>
+			<MessageWrapper>
+				{ interpolateComponents( {
+					mixedString: __(
+						'{{customizationOptions}}Where are customization options?{{/customizationOptions}} In the new checkout experience, payment method details are automatically displayed in your customers’ languages so you don’t have to worry about writing them manually.',
+						'woocommerce-gateway-stripe'
+					),
+					components: {
+						customizationOptions: <b></b>,
```

Comment:
(minor)
```suggestion
						customizationOptions: <b />,
```

or you could even make this more explicit for translators by renaming the `customizationOptions` tag to just `strong`